app/models.py

```diff
@@ -1347,9 +1346,9 @@ class Notification(db.Model):
 
     client_reference = db.Column(db.String, index=True, nullable=True)
 
-    international = db.Column(db.Boolean, nullable=False, default=False)
+    international = db.Column(db.Boolean, nullable=True, default=False)
```

Comment:
Is there protection in place to capture null (an empty string) where this code is called/used?

Reply:
How does changing this to `nullable=True` address the issue raised with the unexpected output?